src/components/Tab/TabInfo.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC } from 'react';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tab: Tab,
+  isActive: boolean,
+  onTabSelected: (tab: Tab) => void,
+}
+
+export const TabInfo: FC<Props> = ({
+  tab,
+  isActive,
+  onTabSelected,
+}) => {
+  const handleTabClick = () => {
+    if (!isActive) {
+      onTabSelected(tab);
+    }
+  };
+
+  return (
+    <li
+      className={isActive ? 'is-active' : ''}
```

Comment:
would you check once more about `li` tag? I have `ul` list and I left this tag in the upper component. Every `li` tag is a part of the tab. So if I move it to the upper component it seems like it will lose sense. Because in TabInfo will be only `a` tag